views/pools/pools-card/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { DefaultTokenSVG, IPXRoundedSVG } from '@/svg';
+
+import CardTradeInfo from './card-trade-info';
+import PoolCardHeader from './pool-card-header';
+import TokenInfo from './token-info';
+
+const PoolCard: FC = () => {
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      justifyContent="center"
+      alignItems="center"
+      gap="0.5rem"
+      bg="lowestContainer"
+      borderRadius="1rem"
+      border="0.063rem solid #C6C6CA"
+      width={['100%', '100%', '100%', '21rem']}
+      p="1rem"
+      transition="all 300ms ease-in-out"
+      nHover={{
+        border: 'none',
+        boxShadow: '0px 24px 46px -10px rgba(13, 16, 23, 0.16)',
+        '.arrow-wrapper': {
```

Comment:
talk to @git-marcopitra  about this aproach instead `State` to control the visibility

Reply:
I agree with this approach, cuz it doesn't fire a rerender